<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doctor</class>
 <widget class="QWidget" name="Doctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doctor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit
{
    border: 2px solid rgb(133, 149, 255);
    border-radius: 10px;
    padding: 10 8px;
    background: #fff;
    selection-background-color: darkgray;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>239</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="patientNewpushButton">
      <property name="text">
       <string>New</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="patientEditpushButton">
      <property name="text">
       <string>Edit</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="patientDeletepushButton">
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>16</width>
     <height>929</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTreeWidget" name="patienttreeWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>33</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Chart No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Visit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Birth</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>First Visit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mobile</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Addr</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>11</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>환자 정보</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>20</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="Patient_Data_widget" native="true">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>400</y>
     <width>761</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
      <item>
       <widget class="QLabel" name="Patient_Data_label">
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>788</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fileButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QOpenGLWidget" name="patientDataupperopenGLWidget"/>
      </item>
      <item>
       <widget class="QOpenGLWidget" name="patientDataloweropenGLWidget"/>
      </item>
      <item>
       <widget class="QOpenGLWidget" name="patientDataocclusionopenGLWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>242</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="PatintInfo_label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Patlent Info</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="patientcomboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="patientBirthdatlineEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>126</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Birthday</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>62</y>
     <width>426</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>11</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="Patlent_Name_label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_ChartNumber_label">
        <property name="text">
         <string>Chart Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_Birthday_label">
        <property name="text">
         <string>Birthday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_Gender_label">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_FirstVist_label">
        <property name="text">
         <string>First Vist</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_LastVist_label">
        <property name="text">
         <string>Last Vist</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="patientNamelineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="baseSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
 width:50px;
 height:15px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="patientChartNumberlineEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
 height: 10px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Chart Number</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="patientGenderMaleradioButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton
{
	background-color:#fff;
	width: 100px;
    height: 30px;
}

QRadioButton::indicator 
{
    background-color: ff3399;
    border-style: outset;
    width: 0px;
    height: 0px;
}

QRadioButton::checked {
   background:#006 ;
 color:#fff;
}

QRadioButton::hover {
  background:#006 ;
 color:#fff;
}</string>
          </property>
          <property name="text">
           <string>Male</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="patientGenderFemaleradioButton">
          <property name="styleSheet">
           <string notr="true">QRadioButton
{
	background-color:#fff;
	width: 100px;
    height: 30px;
}

QRadioButton::indicator 
{
    background-color: ff3399;
    border-style: outset;
    width: 0px;
    height: 0px;
}

QRadioButton::checked {
   background:#006 ;
 color:#fff;
}

QRadioButton::hover {
  background:#006 ;
 color:#fff;
}</string>
          </property>
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="patientGenderNoneradioButton">
          <property name="styleSheet">
           <string notr="true">QRadioButton
{
	background-color:#fff;
	width: 100px;
    height: 30px;
}

QRadioButton::indicator 
{
    background-color: ff3399;
    border-style: outset;
    width: 0px;
    height: 0px;
}

QRadioButton::checked {
   background:#006 ;
 color:#fff;
}

QRadioButton::hover {
  background:#006 ;
 color:#fff;
}</string>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDateEdit" name="patientFirstVistdateEdit">
        <property name="styleSheet">
         <string notr="true">QDateEdit {
	border: 2px solid lightgray;
	border-radius: 10px;
    width:190px;
    height:40px;
}</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="patientLastVistdateEdit">
        <property name="styleSheet">
         <string notr="true">QDateEdit {
	border: 2px solid lightgray;
	border-radius: 10px;
    width:190px;
    height:40px;
}</string>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>12</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1758</year>
          <month>9</month>
          <day>12</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>1282</x>
     <y>171</y>
     <width>16</width>
     <height>42</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>60</y>
     <width>421</width>
     <height>240</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="Patlent_Mobile_label">
        <property name="text">
         <string>Mobile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_phone_label">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_Email_label">
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_Address_label">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Patlent_DetailAddress_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="patientMobilelineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Mobile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="patientPhonelineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="patientEmaillineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>E-mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>@</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="patientEmaillineEdit2"/>
        </item>
        <item>
         <widget class="QComboBox" name="Patient_comboBox">
          <item>
           <property name="text">
            <string>naver.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gmail.com</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="patientAddresslineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="patientDetailAddresslineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Detail Address</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>line_2</zorder>
  <zorder>patienttreeWidget</zorder>
  <zorder>label</zorder>
  <zorder>line_3</zorder>
  <zorder>comboBox</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>comboBox_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
